styles.css

```diff
@@ -0,0 +1,1240 @@
+* {
+    box-sizing: border-box;
+    margin: 0%;
```

Comment:
This is a personal preference of mine, but I prefer not to set the `margin: 0;` and `padding:0` using the `*` selector. The reason being is that it will remove the margin/padding from every element on the page. 

Personally, I prefer to remove the margin/padding from the html/body elements. That way, you get rid of the initial padding/margin from the page, but also keep the default margin/padding on elements such as the `p` tag (there are many other elements with default padding/margin, but p is just an example)